public/static/js/projects.js

```diff
@@ -1110,6 +1144,11 @@ mciModule.controller(
       return $scope.validPatchDefinition(alias) && alias.alias;
     };
 
+    $scope.validContainerSize = function (container_size) {
```

Comment:
I know the save button already checks if they're integers, but should it instead check earlier when adding the container size that CPU/memory are non-negative ints?